### ISSUE TITLE
Fix the status of the plugin manager at the end of integration of int-4

### ISSUE THREAD

**Issue by tsoenen** [sonata-nfv/son-tests#290]
If Integration test 5 is run directly after integration test 4, the start state of the platform for integration test 5 is corrupted. There are no plugins registered with the plugin manager, while at least the SMR, SLM and IA should be registered. This causes integration test 5 to fail. 

If we run integration test 5 on a clean environment, the test succeeds. As temporarily fix, we changed the chain of integration tests, so that integration test 5 is run on a clean environment.

In the long run, we should fix the issue itself. It should not be possible that the platform reaches the corrupted state it has at the end of integration test 4 (this state is not necessarily caused by int-4, it can be caused by an earlier test as well). Therefor, we should investigate how we reached this state and prevent it from happening again.

**Comment by jbonnet** [comment]
@tsoenen 
As a rule, the order of running the tests should be irrelevant (there are even some testing frameworks that force this). To facilitate/achieve this, there's usual a set-up action, ran before each test, that puts the component to be tested in a know state.

Can't we do it this way?

**Comment by tsoenen** [comment]
@jbonnet 

Yes, that is basically what this quick fix does. It is not as clean as your suggestion (where the cleaning of the platform is done at the start of the test, instead of running int-9 to provide a clean platform), but it fixes the test for now. 

Next to making sure that the test starts from a known state, we should also address the real issue that causes this failure: it is somehow possible that our platform reaches a state where the plugins are no longer registered in the plugin manager. Whether this is caused by some standard actions that a owner/customer could do, or by some 'sonata-team admin' moves during the previous test should be checked. But if its the first, then we need to prevent this from happening. This issue shoudl remind us to fix this when we have more time (as I don't think this has the highest priority)
